tests/unit/data/old/config_out.json

```diff
@@ -20,15 +21,17 @@
           "y": "200",
           "width": 200,
           "height": 100,
-          "uid": "Zone 1"
+          "uid": "Zone 1",
+          "align": "center"
```

Comment:
@itsjeyd nope, the thing is, this particular test checks how DnDv2 parses old data format (aka v1) and how it responds to various fields/data members missing/having different names.

`config_out` files represent DnDv2 response to get_configuration handler as it is passed to frontend, so we expect it to have a most recent format (in this case v2.1)